### ISSUE TITLE
Exposing privileged or cross-origin callable is prohibited

### ISSUE THREAD

**Issue by Rudloff** [marcelduran/yslow#133]
On Firefox 36.0a2 with Firebug 2.0.7, I get this error when I click on _Run Test_:

```
Error: Exposing privileged or cross-origin callable is prohibited panel.html:1:11
```

**Comment by webbird** [comment]
Same here. I just upgraded to FF 36.0 and ySlow stopped working.

**Comment by dstutzman-rlo** [comment]
yep, seems this made it to the Stable channel with the release of Firefox 36... It's breaking another one of my extensions

**Comment by grein46087** [comment]
So I have the same issue since upgrading to Firefox 36,  but it impacts my plugin,  Does anyone have a solution around this.

**Comment by dstutzman-rlo** [comment]
yeah, don't use __exposedProps__

https://bugzilla.mozilla.org/show_bug.cgi?id=1065186
https://bugzilla.mozilla.org/show_bug.cgi?id=1082450
